alignak_backend/app.py

```diff
@@ -206,6 +206,238 @@ def pre_get(resource, user_request, lookup):
                                        {'_realm': {'$in': resources_get_custom[resource]}}]}]
 
 
+# Log checks results
+def pre_logcheckresult_post(items):
+    """
+    Hook before adding new forcecheck
+
+    :param items: realm fields
+    :type items: dict
+    :return: None
+    """
+    hosts_drv = current_app.data.driver.db['host']
+    for dummy, item in enumerate(items):
+        # Set _realm as host's _realm
+        host = hosts_drv.find_one({'_id': item['host']})
+        item['_realm'] = host['_realm']
+
+
+def after_insert_logcheckresult(items):
+    """
+    Hook after logcheckresult inserted.
+
+    :param items: realm fields
+    :type items: dict
+    :return: None
+    """
+    for dummy, item in enumerate(items):
+        # Create an history event for the new forcecheck
+        data = {
+            'date': datetime.utcnow(),
+            'host': item['host'],
```

Comment:
So we will have _date_ = _created_ 
Perhaps _date_ fill with the date of the check